app/views/admin/shared/_nested_sidebar.html.erb

```diff
@@ -1,55 +1,45 @@
-<% menu_items.each do |group_name, group| %>
-  <li>
-    <% if group[:children].length == 1 %>
-      <% if group[:children][0][:visible] %>
-        <a class="crayons-link crayons-link--block <%= "crayons-link--current" if deduced_scope(request) == group[:children][0][:controller] %>"
-           href="/admin/<%= group[:children][0][:controller] %>"
-           aria-current="<%= "page" if deduced_controller(request) == group[:children][0][:controller] %>"
-           data-action="click->sidebar#expandDropdown">
-          <%= crayons_icon_tag(group[:svg], class: "dropdown-icon", aria_hidden: true) %>
+<nav>
+  <ul class="m-0">
+    <li>
+      <a class="c-link c-link--block c-link--icon-left" href="<%= admin_path %>" aria-current="<%= "page" if controller.controller_name == "overview" %>">
+        <%= crayons_icon_tag("stack-line", class: "c-link__icon") %>
+        Overview
+      </a>
+    </li>
+
+    <% menu_items.each do |group_name, group| %>
+      <% current = deduced_scope(request).to_s == group[:children][0][:parent].to_s %>
+      <% children = group[:children].length == 1 %>
+      <% path = children ? group[:children][0][:controller] : "#{group_name}/#{group[:children].detect { |child| child[:visible] }[:controller]}" %>
+      <% path = "#{admin_path}/#{path}" %>
+      <li>
+        <a
+          class="c-link c-link--block c-link--icon-left <%= "c-link--current" if current %>"
+          aria-current="<%= "page" if current && children %>"
+          href="<%= path %>">
```

Comment:
Does the `c-link--current` CSS class indicate this is the active section and if it happens to have child elements, then it's the current page? Just trying to make sure I understand the logic as `[aria-current="page"]` is typically enough to style the link in navigation if that link is for the current page.

Reply:
That is correct, sir.

Reply:
if there's no children, i use aria-current. if there are children, i use css class to visually indicate active prent and then use aria-current for appropriate 2nd level nav link.